### PR TITLE
Add default export to ssm middleware

### PR DESCRIPTION
**Pull request by dbartholomae**
This seems to be an oversight - the other middlewares have it, and ssm not having it breaks TypeScript imports

### CODE REVIEW

**Comment by theburningmonk** [comment]
@dbartholomae you're right! it's an oversight, it was my bad. While you're here, do you mind popping a default export into the secrets-manager package as well?

**Comment by dbartholomae** [comment]
Added :)

File: packages/secrets-manager/index.d.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { SecretsManager } from 'aws-sdk'
 import middy from '../core'
+import ssm from '../ssm'
```
Comment (theburningmonk) [inline packages/secrets-manager/index.d.ts]: is this required? the two should be separate, right?

Comment (dbartholomae) [reply packages/secrets-manager/index.d.ts]: Thanks for catching that! My IDE snuck that in ^^'
I'll remove it and double check the rest.

Comment (dbartholomae) [reply packages/secrets-manager/index.d.ts]: I've removed the line

**Comment by lmammino** [comment]
@theburningmonk are you ok with the latest changes?

**Comment by dbartholomae** [comment]
Accidentially added another commit, will remove it asap

**Pull request merged**